cheatsheets/Secure_Product_Design_Cheat_sheet.md

```diff
@@ -0,0 +1,92 @@
+# Secure Product Design Cheat Sheet
+
+## Introduction
+
+The purpose of Secure Product Design is to ensure that all products meet or exceed the security requirements laid down by the organisation as part of the development lifecycle and to ensure that all security decisions made about the product being developed are explicit choices and result in the correct level of security for the product being developed.
+
+## Methodology
+
+As a basic start, establish secure defaults, minimise the attack surface area, and fail securely to those well-defined and understood, defaults.
+
+Secure Product Design comes about through two processes:
+
+1. **_Product Inception_**; and
+2. **_Product Design_**
+
+The first process happens when a product is conceived, or when an existing product is being re-invented. The latter is continuous, evolutionary, and done in the agile way, close to where the code is being written.
```

Comment:
```suggestion
The first process happens when a product is conceived, or when an existing product is being re-invented. The latter is continuous, evolutionary, and done in an agile way, close to where the code is being written.
```

Reply:
Maybe we need a reference to the DevSecOps CS here? (I found that [here](https://github.com/dominikdesmit/owasp-devsecops-cheat-sheet). Not sure why it's not with the rest of the cheat sheets.)

The reason that I say that is security + agile is not intuitive and takes a lot of hard thinking to get right. Most agile efforts that I've seen only get as far as having a box in a PowerPoint diagram labeled 'security' and a couple arrows going in and out of it. And if you drill down on that security box, you get a whole lot of "magic happens here" and it's obvious that they haven't thought about it at all. So, let's help them get there.